transformers/integration_tests/cat_file_test.go

```diff
@@ -36,168 +37,212 @@ import (
+				Transformer: flip.Transformer{},
+			}
+
+			t := initializer.NewTransformer(db)
+
+			logs, err := logFetcher.FetchLogs(
+				[]common.Address{common.HexToAddress(catFileConfig.ContractAddresses[0])},
+				[]common.Hash{common.HexToHash(catFileConfig.Topic)},
+				header)
+			Expect(err).NotTo(HaveOccurred())
+
+			eventLogs := test_data.CreateLogs(header.Id, logs, db)
+
+			err = t.Execute(eventLogs)
+			Expect(err).NotTo(HaveOccurred())
+
+			var dbResult catFileFlipModel
+			err = db.Get(&dbResult, `SELECT ilk_id, what, flip FROM maker.cat_file_flip`)
+			Expect(err).NotTo(HaveOccurred())
+
+			ilkID, err := shared.GetOrCreateIlk("0x4554482d41000000000000000000000000000000000000000000000000000000", db)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(dbResult.Ilk).To(Equal(strconv.FormatInt(ilkID, 10)))
+			Expect(dbResult.What).To(Equal("flip"))
+			Expect(dbResult.Flip).To(Equal(test_data.FlipEthV100Address()))
+		})
+
+		It("persists a vow event", func() {
+			vowBlockNumber := int64(8928165)
+			header, err := persistHeader(db, vowBlockNumber, blockChain)
+			Expect(err).NotTo(HaveOccurred())
+			catFileConfig.TransformerName = constants.CatFileVowTable
+			catFileConfig.Topic = constants.CatFileVowSignature()
+			catFileConfig.StartingBlockNumber = vowBlockNumber
+			catFileConfig.EndingBlockNumber = vowBlockNumber
+
+			initializer := event.ConfiguredTransformer{
+				Config:      catFileConfig,
+				Transformer: vow.Transformer{},
+			}
+			t := initializer.NewTransformer(db)
+
+			logs, err := logFetcher.FetchLogs(
+				[]common.Address{common.HexToAddress(catFileConfig.ContractAddresses[0])},
+				[]common.Hash{common.HexToHash(catFileConfig.Topic)},
+				header)
+			Expect(err).NotTo(HaveOccurred())
+
+			eventLogs := test_data.CreateLogs(header.Id, logs, db)
+
+			err = t.Execute(eventLogs)
+			Expect(err).NotTo(HaveOccurred())
+
+			var headerID int64
+			err = db.Get(&headerID, `SELECT id FROM public.headers WHERE block_number = $1`, vowBlockNumber)
+			Expect(err).NotTo(HaveOccurred())
+
+			var dbResult catFileVowModel
+			err = db.Get(&dbResult, `SELECT what, data FROM maker.cat_file_vow`)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(dbResult.What).To(Equal("vow"))
+			Expect(dbResult.Data).To(Equal(test_data.VowAddress()))
+		})
 	})
 
-	It("persists a vow event", func() {
-		vowBlockNumber := int64(8928165)
-		header, err := persistHeader(db, vowBlockNumber, blockChain)
-		Expect(err).NotTo(HaveOccurred())
-		catFileConfig.TransformerName = constants.CatFileVowTable
-		catFileConfig.Topic = constants.CatFileVowSignature()
-		catFileConfig.StartingBlockNumber = vowBlockNumber
-		catFileConfig.EndingBlockNumber = vowBlockNumber
-
-		initializer := event.ConfiguredTransformer{
-			Config:      catFileConfig,
-			Transformer: vow.Transformer{},
+	Describe("1.1.0 Cat Contract", func() {
+		var catFileConfig = event.TransformerConfig{
+			ContractAddresses: []string{test_data.Cat110Address()},
```

Comment:
I think this is a good example of how this may be confusing/unclear to future developers - like for testing `cat_file_box` we _have_ to use the 1.1.0 version. But for the other events, we could use either contract, and I'm not sure that this is really clear anywhere in the code. I don't have a good idea on how to clarify this some, but just wanted to point it out.

Reply:
Also, just thinking that maybe it would be good to include tests for the other events, but for 1.1.0. 🤔

Reply:
Totally. I'm confused 🤪  just thinking about this. So for something like `cat_file_vow` that exists in both versions and is unchanged, we won't duplicate those tests for 1.1.0? And since `cat_file_box` was introduced in 1.1.0 we'll only have tests for it there?  We should probably see how this plays out as we make changes.

Reply:
Yeah, we'd end up duplicating tests for 1.1.0... which isn't ideal either. I'm good with holding off with making that change for now, and seeing how the rest of these changes pan out before making a decision.